example/payload.json

```diff
@@ -1,4 +1,4 @@
 {
 	"type": "pjnz",
-	"path": "src/inst/testdata/Botswana2018.PJNZ"
```

Comment:
Also wondering if there is a better way to work around this in future. Perhaps that will come out of the work setting up docker volume though? Instead of having dupe example payloads, one for when the data is living in docker in a `src` directory and one for when running the plumber API outside of docker.

Reply:
This is where a configurable root directory would come in handy!